### PR TITLE
Add script to update vespa-version MERGEOK

### PR DESCRIPTION
**Pull request by kkraune**
Trig this weekly

### CODE REVIEW

File: screwdriver.yaml [hunk 1]
```diff
@@ -75,3 +75,23 @@ jobs:
...
           $SD_SOURCE_DIR/test/test.py -c $SD_SOURCE_DIR/test/_test_config-large.yml -w $SD_SOURCE_DIR
+
+  update-vespa-version:
+    image: buildpack-deps
+    annotations:
+      screwdriver.cd/buildPeriodically: H H(0-5) * * 3 # some time between 12:00 AM UTC (midnight) to 5:59 AM UTC Wed
+    steps:
+      - update-to-latest: |
+          VESPA_VERSION=$(curl -s 'https://repo1.maven.org/maven2/com/yahoo/vespa/parent/maven-metadata.xml' | \
+            grep '<latest>' | sed 's/<\/*latest>//g; s/ //g;')
```
Comment (aressem) [inline screwdriver.yaml:86]: Or:
`curl -sSL https://repo1.maven.org/maven2/com/yahoo/vespa/parent/maven-metadata.xml|grep -oP '<latest>\K([0-9]+\.[0-9]+\.[0-9]+)'`

**Review by aressem — APPROVED** [review]

File: screwdriver.yaml (hunk 1 above)
Comment (kkraune) [reply screwdriver.yaml:86]: Thanks!

File: screwdriver.yaml [hunk 2]
```diff
@@ -83,7 +83,7 @@ jobs:
     steps:
       - update-to-latest: |
           VESPA_VERSION=$(curl -sSL https://repo1.maven.org/maven2/com/yahoo/vespa/parent/maven-metadata.xml | \
-            grep '<latest>\K([0-9]+\.[0-9]+\.[0-9]+)' | sed 's/<\/*latest>//g; s/ //g;')
+            grep -oP '<latest>\K([0-9]+\.[0-9]+\.[0-9]+)' | sed 's/<\/*latest>//g; s/ //g;')
```
Comment (aressem) [inline screwdriver.yaml:86]: No need to execute the last sed , because the grep will print only the version match.

Comment (kkraune) [reply screwdriver.yaml:86]: Thanks - learned something new there - removed in #2363

**Pull request merged**